apis/v1alpha1/opampbridge_types.go

```diff
@@ -0,0 +1,137 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OpAMPBridgeSpec defines the desired state of OpAMPBridge.
+type OpAMPBridgeSpec struct {
+	// OpAMP backend Server endpoint
+	// +required
+	Endpoint string `json:"endpoint"`
+	// Protocol scheme for the communication between OpAMP server and OpAMP Bridge
+	// +required
+	Protocol string `json:"protocol"`
+	// Capabilities supported by the OpAMP Bridge
+	// +required
+	Capabilities []string `json:"capabilities"`
```

Comment:
Should we make this an Enum / use strong typing here?

Reply:
Done. Enum is created for the Capabilities.